learning_resources_search/serializers.py

```diff
@@ -239,6 +239,7 @@ def to_representation(self, obj):
     "professional",
     "free",
     "learning_format",
+    "delivery",
```

Comment:
Search aggregated by delivery does not work RN you need to add `"delivery": FilterConfig("delivery.code"),` to SEARCH_FILTERS in learning_resource_search/constants.py for delivery to be a valid aggregation option.

Additionally you should add delivery to LearningResourcesSearchRequestSerializer